devices/Mcp7940xx/AlarmMinuteRegister.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Mcp7940xx
+{
+    /// <summary>
+    /// Alarm minute register constants.
+    /// </summary>
+    /// <remarks>
+    /// Used for both Alarm1 and Alarm2 registers.
+    /// </remarks>
+    internal enum AlarmMinuteRegister
```

Comment:
There's only a single element on this enum... wondering if this wouldn't be better a constant...
Same comment for all the other "single" item enums that follow, and there are quite a lot! 😅

Reply:
I agree, they are a bit empty. I mostly was keeping the pattern I saw used with other devices for consistency for future maintainers. If we do shift these out from the enums, where would they make the most sense to live? Inside of Mcp7940m or create a register-constant class to hold them?

Reply:
I would go with that "storage" class for registers. This is something that can be found on other bindings so it's consistent with the design patterns being used here.